guides/common/modules/proc_configuring-logging-to-journal.adoc

```diff
@@ -23,3 +14,28 @@ endif::[]
 --foreman-logging-type journald \
 --foreman-proxy-log JOURNAL
 ----
+
+Journal now forwards log messages to `rsyslog`, and `rsyslog` writes the log messages to `/var/log/messages`.
+After this change, the log messages do not appear in `/var/log/foreman/production.log` or `/var/log/foreman-proxy.log`.
+
+.Procedure for configuring file-based logging
+. Enter the following `{foreman-installer}` command to configure the default file-based logging:
```

Comment:
```suggestion
* Enter the following `{foreman-installer}` command to configure the default file-based logging:
```

convention in Foreman documentation: If the procedure only has one action item, it's an unordered list.